### PR TITLE
Include more in the event handler status

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Include more in the event handler status
- Reuse existing ip addresses in event handler

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1290

### CODE REVIEW

File: event-handler/event-channel-manager.js [hunk 1]
```diff
@@ -23,12 +23,11 @@ class EventChannelManager {
...
   }
 
-  async getNetworkInfo() {
-    const ipAddr = await getIpAddress(NodeConfigs.HOSTING_ENV === HostingEnvs.COMCOM || NodeConfigs.HOSTING_ENV === HostingEnvs.LOCAL);
+  getNetworkInfo() {
+    const ipAddr = (NodeConfigs.HOSTING_ENV === HostingEnvs.COMCOM || NodeConfigs.HOSTING_ENV === HostingEnvs.LOCAL) ? this.node.ipAddrInternal : this.node.ipAddrExternal;
     const eventHandlerUrl = new URL(`ws://${ipAddr}:${NodeConfigs.EVENT_HANDLER_PORT}`);
     return {
       url: eventHandlerUrl.toString(),
-      port: NodeConfigs.EVENT_HANDLER_PORT,
```
Comment (jiyoung-an) [inline event-handler/event-channel-manager.js:31]: Q. Could it be omitted because the port is fixed to 443?

**Review by jiyoung-an — APPROVED** [review]
LGTM with a question! Thanks!

**Comment by platfowner** [comment]
Thanks for the review! @jiyoung-an Now merging..

File: event-handler/event-channel-manager.js (hunk 1 above)
Comment (platfowner) [reply event-handler/event-channel-manager.js:31]: ah, just removed as it's included in the url.

**Pull request merged**